File: BackEnd/backend/src/main/java/com/fitapp/backend/application/dto/routine/response/RoutineStatisticsResponse.java
package com.fitapp.backend.application.dto.routine.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class RoutineStatisticsResponse {
    private long totalRoutines;
    private long activeRoutines;
    private long inactiveRoutines;
}

File: FrontEnd/app/src/main/java/com/fitapp/appfit/response/routine/response/RoutineStatisticsResponse.kt
package com.fitapp.appfit.response.routine.response


import com.google.gson.annotations.SerializedName

data class RoutineStatisticsResponse(
    @SerializedName("totalRoutines") val totalRoutines: Long,
    @SerializedName("activeRoutines") val activeRoutines: Long,
    @SerializedName("inactiveRoutines") val inactiveRoutines: Long
)